packages/salesforcedx-vscode-expanded/README.md

```diff
@@ -8,17 +8,17 @@ This extension pack includes tools for developing on the Salesforce platform in
 
 Before you set up Salesforce Extensions for VS Code, make sure that you have these essentials.
 
-- **Salesforce CLI**  
+- **Salesforce CLI**
   Before you use Salesforce Extensions for VS Code, [set up Salesforce CLI](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup).
 - **A Salesforce DX project**
-  Open your Salesforce DX project in a directory that contains an `sfdx-project.json` file. Otherwise, some features don’t work.  
+  Open your Salesforce DX project in a directory that contains an `sfdx-project.json` file. Otherwise, some features don’t work.
   If you don't already have a Salesforce DX project, create one with the **SFDX: Create Project** command (for development against scratch orgs) or the **SFDX: Create Project with Manifest** command (for development against sandboxes or DE orgs). Or, see [create a Salesforce DX project](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_workspace_setup.htm) for information about setting up a project using Salesforce CLI.
-- **Java Platform, Standard Edition Development Kit**  
+- **Java Platform, Standard Edition Development Kit**
   Some features in Salesforce Extensions for VS Code depend upon the Java Platform, Standard Edition Development Kit (JDK). You need to have either version 8 or version 11 of the JDK installed.
 
-  If you don’t already have version 8 or 11 of the JDK installed, you can install the latest version of the Java 8 JDK from [Java SE Development Kit 8 Downloads](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) or the latest version of the Java 11 JDK from [Java SE Development Kit 11 Downloads](https://www.oracle.com/technetwork/java/javase/downloads/jdk11-downloads-5066655.html).
+  If you don’t already have version 8, 11, or 17 of the JDK installed, you can install the latest version of the Java 8 JDK from [Java SE Development Kit 8 Downloads](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html), or the latest version of the Java 11 JDK from [Java SE Development Kit 11 Downloads](https://www.oracle.com/technetwork/java/javase/downloads/jdk11-downloads-5066655.html), or Java 17 from [Java Downloads](https://www.oracle.com/java/technologies/downloads/#java17).
```

Comment:
NIt: Looks like java8 and java11 are on the same page as the new java 17 link.  Should we update those links as well? https://www.oracle.com/java/technologies/downloads/#java8 https://www.oracle.com/java/technologies/downloads/#java11

Reply:
Let's have @AnanyaJha weigh in.  My initial thoughts when working on this was to not touch anything that pertained to the existing Java 8 or Java 11 docs, and only make changes for Java 17.  I think it's a good idea though, so if Ananya says it's OK, I'll make the changes.

Reply:
Sure that sounds good to me

Reply:
Done